### ISSUE TITLE
Warns "Warning:  property is not valid" for `@require` statements

### ISSUE THREAD

**Issue by awayken** [rossPatton/stylint#35]
When the "valid" flag is set to true, `@require` statements are warned as being invalid properties. To recreate the issue, create the following files and run the command at the bottom.

base.styl
```stylus
body {
    color: red;
}
```

site.styl
```stylus
@require 'base';
```

config.json
```json
{
    "alphabetical": false,
    "borderNone": false,
    "brackets": false,
    "colons": false,
    "commaSpace": false,
    "commentSpace": false,
    "cssLiteral": false,
    "depthLimit": false,
    "duplicates": false,
    "efficient": false,
    "enforceBlockStyle": false,
    "enforceVarStyle": false,
    "extendPref": false,
    "globalDupe": false,
    "indentSpaces": false,
    "leadingZero": false,
    "maxWarnings": false,
    "maxWarningsKill": false,
    "mixed": false,
    "namingConvention": false,
    "parenSpace": false,
    "placeholders": false,
    "semicolons": false,
    "Whitespace": false,
    "universal": false,
    "valid": true,
    "zeroUnits": false,
    "zIndexDuplicates": false,
    "zIndexNormalize": false
}
```

Then execute this:
```
$ stylint site.styl -c config.json
Warning:  property is not valid
File: site.styl
Line: 1: @require 'base';
```

**Comment by rossPatton** [comment]
thanks for filing these, i have a feeling both of these have the same root cause. will take a look this weekend.

**Comment by rossPatton** [comment]
this should be fixed in the latest version, 0.8.12
